frontend/src/domain/entities/QuestionDifficulty.ts

```diff
@@ -1 +1 @@
-export type QuestionDifficulty = "Easy" | "Medium" | "Hard";
+export type QuestionDifficulty = "All" | "Easy" | "Medium" | "Hard";
```

Comment:
I think adding 'All' here is a bit misleading since it entails that I can create a question that has Easy, Medium and Hard difficulty. I initially added this type to ensure that IQuestionInput only takes in these 3 difficulties

I get that you need it for the questions filter though.

Reply:
Ok, will remove because it makes sense it doesn't belong here. This means I cannot use your difficultyOptions for filter though.